FrontMen/fm-hours
# Feat: remove create employee

# Changes

## Related issues

Resolves #166 

## Added

- Default avatar picture

## Removed

- `createEmployee` Firebase Function

## Changed

- First login logic now handled on client side

## How to test

- Handle all "first time login" flow in the onAuthStateChanged action
- Remove the if statement with a sleep function in it
- Make sure that the function is removed from the Firebase
- Add logic to prevent more than one employee with the same email
- Keep compatibility with already existing employees
- Only employees that have a account manually created by an admin can login

## Screenshots

N/A
